lib/Molmed/Sisyphus/Common.pm

```diff
@@ -730,14 +731,26 @@ sub saveMd5{
     my $absFile = $file;
     $file =~ s:^$rfPath/::;
     my $runfolder = basename($rfPath);
-
+    
+    if (my $lock = new File::NFSLock {
+        file      => $file,
```

Comment:
aren't you actually locking the wrong file here? this is the file the calculated checksum refers to but you want to lock the  `$rfPath/MD5/sisyphus.md5`file, right?